uint16 setPoint
uint16 currentValue
bool maximumReached
bool ledVal
